app/models/shipit/anonymous_user.rb

```diff
@@ -1,5 +1,9 @@
 module Shipit
   class AnonymousUser
+    def serializer_class
+      UserSerializer
+    end
```

Comment:
What do you think @byroot ? I'd rather not have it in here but compared to making this inherit from `User` I prefer this solution.

Actually I'm slightly confused about how serialization worked previously for this so I'll look into it more

Reply:
In general I'm not a huge fan of this AMS API, but it's fine. 

I'm like to get rid of AMS anyway.